### PR TITLE
Change add plan button text on manage plans page

### PR DESCRIPTION
**Pull request by moshthepitt**
Fix: #418

### CODE REVIEW

File: src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ const PlanDefinitionList = (props: PlanListProps) => {
   const { fetchPlans, plans, service } = props;
   const [loading, setLoading] = useState<boolean>(true);
 
-  const apiService = new service('/plans');
+  const apiService = new service('plans');
```
Comment (p-netm) [inline src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx]: `export const OPENSRP_PLANS = 'plans';` This is trivial but maybe we should also use constants for the api endpoints.

File: src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx [hunk 2]
```diff
@@ -99,7 +99,10 @@ const PlanDefinitionList = (props: PlanListProps) => {
           <h3 className="mt-3 mb-3 page-title">{pageTitle}</h3>
         </Col>
         <Col md={4}>
-          <LinkToNewPlans classProp="focus-investigation btn btn-primary float-right mt-3 mb-3" />
+          <LinkToNewPlans
+            className="focus-investigation btn btn-primary float-right mt-3 mb-3"
+            text="Add Plan"
```
Comment (cKellyDesign) [inline src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx]: @moshthepitt let's make `Add Plan` a constant, ama?

File: src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx (hunk 1 above)
Comment (cKellyDesign) [reply src/containers/pages/InterventionPlan/PlanDefinitionList/index.tsx]: Agreed, let's move endpoint strings like this into constants :)

**Comment by moshthepitt** [comment]
@cKellyDesign @p-netm please re-review.

File: src/constants.tsx [hunk 3]
```diff
@@ -138,6 +138,10 @@ export const NAME = 'Name';
 export type NAME = typeof NAME;
 export const NEW_PLAN = 'New Plan';
 export type NEW_PLAN = typeof NEW_PLAN;
+export const ADD_PLAN = 'Add Plan';
+export type ADD_PLAN = typeof ADD_PLAN;
+export const UPDATE_PLAN = 'Update Plan';
+export type UPDATE_PLAN = typeof UPDATE_PLAN;
```
Comment (cKellyDesign) [inline src/constants.tsx]: 👍

**Pull request merged**